probe-rs/src/bin/probe-rs/cmd/cargo_embed/rttui/channel.rs

```diff
@@ -22,6 +23,7 @@ pub struct ChannelConfig {
     pub name: Option<String>,
     pub up_mode: Option<ChannelMode>,
     pub format: DataFormat,
+    pub socket: Option<String>,
```

Comment:
Could we use `std::net::SocketAddr` here directly? It implements `FromStr`, so I think it should just work.

Reply:
I changed it to `Socketaddr`, but now it does not accept `localhost:12345` as config, only actual ip addresses `127.0.0.1:12345`. Besides that it still works fine. Do you think this is preferred?

Reply:
I guess it is better as the parsing of the config is more strict with SocketAddr